src/Components/DetailItem/DropDownBox.js

```diff
@@ -0,0 +1,153 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
+import './DropDownBox.scss';
+import OrderBox from './OrderBox';
+
+class DropDownBox extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      number: 0,
+      value: '',
+      size: '',
+      selectPrice: 16200,
+      defaultPrice: 0,
+      click: false,
+      click2: false,
+      item: [],
+      orderBox: [],
+      priceBox: 0,
+    };
+  }
+
+  componentDidMount() {
+    this.setState({});
+  }
+  // 상품 클릭시 클릭 바뀌게
+  handleChange = (el) => {
+    const { click, number } = this.state;
+    this.setState({ number: number + 1, value: el.label });
+  };
+
+  //사이즈 클릭시 값이 누적되고 오더박스 카드가 생김
+  handleSizeCheck = (e) => {
+    // console.log('e는 뭐지?', e);
+    const { click2, selectPrice, item, value, click, orderBox } = this.state;
+    let arr = [...item];
+    arr.push(selectPrice);
+    // console.log('arr에 item[] 복사됨 : ', arr);
+    let totalPrice = arr.reduce((acc, obj) => acc + obj, 0);
+    // console.log('totalPrice에 값이 더해짐 :', totalPrice);
+    let orderBox2 = [...orderBox];
+    orderBox2.push({ value, size: e.value, priceBox: 16200 });
```

Comment:
아래와 같이 적어주시면 한번에 적어주실 수 있습니다. 
```suggestion
      let orderBox2 = [...orderBox, { value, size: e.value, priceBox: 16200 }];
```

Reply:
저렇게 적용할 수가 있네요!
지금은 코드가 많이 바뀌어서 다르게 적용했습니다.
다음엔 저렇게 적용해보겠습니다!